### DISCUSSION TITLE
Is the beta now the release?

### DISCUSSION THREAD

**Discussion by Procyon-b** [openstyles/stylus#1536] [General]
(using chrome)
Looking at my log, up until version 1.5.25 the date of the "release" matches the update of the normal version (`clngdb...`). But with version 1.5.26 the release's date matches the update of the beta version (`apmmpa...`). With the latest release (v1.5.29) `(Beta)` has been removed from the name of the beta extension.
Should we consider now the beta as the normal version, and the normal version as a stable version?

**Comment by Mottie** [comment]
The "Beta" part of the extension wasn't included in the last update, but it is still the beta version. I've been pushing the latest work to the beta Chrome Web Store and the previous release everywhere else.

**Comment by Procyon-b** [comment]
I mean that previously the latest released version https://github.com/openstyles/stylus/releases was always the normal version. But now it is the beta.
